### PR TITLE
Run ci checks for ✨Source Auth0: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #28928: Auth0

### PR DESCRIPTION
**Pull request by sajarin**
## What
Run ci checks for #28972 

*Migrating Source Auth0 to Low-Code CDK*
Closes #28928

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>
Full Test Results

image

</details>

## 🚨 User Impact 🚨
- No breaking changes, migration to low-code
<details><summary><strong>Migrate Connector: Source Auth0</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-auth0 test report (commit [`5468adbf49`](https://github.com/airbytehq/airbyte/pull/29336/commits/5468adbf49b1cb2ccce92ce5ac530bcc0c7ab311)) - ❌

⏲️  Total pipeline duration: 301mn29s 

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-auth0/metadata.yaml | ✅        |
| Connector version semver check                                      | ✅        |
| Connector version increment check                                   | ✅        |
| QA checks                                                           | ✅        |
| Code format checks                                                  | ✅        |
| Connector package install                                           | ✅        |
| Build source-auth0 docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                    | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/saj_auth0-low/1691688770/5468adbf49b1cb2ccce92ce5ac530bcc0c7ab311/source-auth0/0.4.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5468adbf49b1cb2ccce92ce5ac530bcc0c7ab311)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-auth0 test
```

File: airbyte-integrations/connectors/source-auth0/source_auth0/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['credentials']['access_token'] }}"
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
```
Comment (vasilisgav) [inline airbyte-integrations/connectors/source-auth0/source_auth0/manifest.yaml:23]: Hi @sajarin is this paginator able to handle auth0 pagination?

there is an open PR (for a bug resolution) about it https://github.com/airbytehq/airbyte/pull/29359 and wondering how the pagination will be handled by the new low-code CDK.
Cheers

**Comment by octavia-squidington-iii** [comment]
##  source-auth0 test report (commit [`d477633c57`](https://github.com/airbytehq/airbyte/pull/29336/commits/d477633c57cd6cdfe99e606e50d77ad53c08fd44)) - ❌

⏲️  Total pipeline duration: 301mn14s 

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-auth0/metadata.yaml | ✅        |
| Connector version semver check                                      | ✅        |
| Connector version increment check                                   | ✅        |
| QA checks                                                           | ✅        |
| Code format checks                                                  | ✅        |
| Connector package install                                           | ✅        |
| Build source-auth0 docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                    | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/saj_auth0-low/1691780037/d477633c57cd6cdfe99e606e50d77ad53c08fd44/source-auth0/0.4.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:d477633c57cd6cdfe99e606e50d77ad53c08fd44)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-auth0 test
```

**Comment by octavia-squidington-iii** [comment]
##  source-auth0 test report (commit [`99a449977d`](https://github.com/airbytehq/airbyte/pull/29336/commits/99a449977d0f664a7479aba891d9f7378a10e76a)) - ❌

⏲️  Total pipeline duration: 300mn36s 

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-auth0/metadata.yaml | ✅        |
| Connector version semver check                                      | ✅        |
| Connector version increment check                                   | ✅        |
| QA checks                                                           | ✅        |
| Code format checks                                                  | ✅        |
| Connector package install                                           | ✅        |
| Build source-auth0 docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                    | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/saj_auth0-low/1691798238/99a449977d0f664a7479aba891d9f7378a10e76a/source-auth0/0.4.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:99a449977d0f664a7479aba891d9f7378a10e76a)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-auth0 test
```

**Comment by octavia-squidington-iii** [comment]
##  source-auth0 test report (commit [`e74adb23cb`](https://github.com/airbytehq/airbyte/pull/29336/commits/e74adb23cbed591eb3a9d2b6556ba9cca41ee10f)) - ❌

⏲️  Total pipeline duration: 301mn32s 

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-auth0/metadata.yaml | ✅        |
| Connector version semver check                                      | ✅        |
| Connector version increment check                                   | ✅        |
| QA checks                                                           | ✅        |
| Code format checks                                                  | ✅        |
| Connector package install                                           | ✅        |
| Build source-auth0 docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                    | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/saj_auth0-low/1692131141/e74adb23cbed591eb3a9d2b6556ba9cca41ee10f/source-auth0/0.4.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:e74adb23cbed591eb3a9d2b6556ba9cca41ee10f)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-auth0 test
```

**Pull request closed without merging**